packages/core/src/util/normalizeConfig.js

```diff
@@ -162,6 +166,22 @@ function normalizeServiceName(config) {
   }
 }
 
+/**
+ * @param {InstanaConfig} config
+ */
+function normalizePackageJsonPath(config) {
+  if (config.packageJsonPath == null && process.env['INSTANA_PACKAGE_JSON_PATH']) {
```

Comment:
Can we simplify the check by `!config.packageJsonPath` instead of `config.packageJsonPath == null`